### DISCUSSION TITLE
Retroarch Default Config

### DISCUSSION THREAD

**Discussion by Rom0067** [EmulatorJS/EmulatorJS#851] [Q&A]
Hello I am fairly familiar with retroarch and was wondering how to make changes to the default retroarch configuration.  I can't figure out how to change global settings only settings when I'm in a game.  I would like to change some default options so when users log in they do not need to make any changes.  My users are technically illiterate and walking them through retroarch is something I would rather not do.  I have emulatorjs installed via docker and it is overall working fantastically.  I apologize in advance if I'm dumb and just missed something.  Thank you!

**Comment by ethanaobrien** [comment]
This EmulatorJS project has no official docker image. This project is a tool for developers to build websites/rom managers/etc with, and is nothing more than a webassembly emulator by itself. Would you try the demo out at https://demo.emulatorjs.org to verify you’re in the right place?

**Reply by Rom0067** [reply]
Apologies I think I am in the wrong place...
